controllers/imageupdateautomation_controller.go

```diff
@@ -793,3 +787,24 @@ func (r *ImageUpdateAutomationReconciler) recordSuspension(ctx context.Context,
 		r.MetricsRecorder.RecordSuspend(*objRef, auto.Spec.Suspend)
 	}
 }
+
+// TemplateMsg renders a msg template, returning the message or an error.
+func TemplateMsg(messageTemplate string, templateValues *TemplateData) (string, error) {
```

Comment:
Does this need to be exported? I think it's better not to, by default. (You can accuse me of hypocrisy, I've probably exported random funcs elsewhere in the controller code ...)

Reply:
Thanks! Definitely shouldn't be exported